docs/4-Transition/lessons.md

```diff
@@ -47,13 +47,100 @@ Earlier versions of FileNet have the filesystem set to read only by default. How
+
+![FileNet Lessons Learn 7.png](https://zenhub.ibm.com/images/6442f46ac0371b5acaba3fc4/0e96fca8-9247-4fe3-9979-5362b8046bb5)
+
+Now hit the "Record Network Log" button again to start recording network activity (The Record Network Log will have a red square when recording). Next enable the "Preserve log" option. If done correctly Chrome should now display an empty log with "Record Network Log" and "Preserve log" enabled. 
+
+![FileNet Lessons Learn 8.png](https://zenhub.ibm.com/images/6442f46ac0371b5acaba3fc4/df17983c-42ae-4759-bfdb-9346f78e8299)
+
+Now go back to the ACCE console. We will recreate the error for the logs. So we need to create a document with content. Once we get the error message again, we can examine the Chrome debugging window again. We can turn off "Record Network Log". Scrolling through the header, there is a file with an error. Here we saw the error was "413 Payload Too Large". 
+
+![FileNet Lessons Learn 9.png](https://zenhub.ibm.com/images/6442f46ac0371b5acaba3fc4/2b397a71-3f88-4617-a85d-17fc67cd3456)
+
+Optionally we could select the export button. This will export the logs to a file. When viewing the file we saw the same "413 Payload Too Large" error.
+
+```
+...
+"response": {
+          "status": 413,
+          "statusText": "Payload Too Large",
+          "httpVersion": "http/2.0",
+...
+}
+```
+
+So in this situation, the NGINX server was rejecting the request due to the body size exceeding the limit.
+
+#### **The Solution**
+
+The way the team resolved this was pretty straight forward. We needed to add an annotation to the FileNet CR within EKS. We added this `nginx.ingress.kubernetes.io/proxy-body-size: "0"` annotation to the `shared_configuration` section of the CR. 
+
+```yaml
+shared_configuration:
+  ## The deployment context as selected. 
+  sc_deployment_context: FNCM
+  show_sensitive_log: true
+  no_log: false 
+  sc_ingress_enable: true 
+  sc_service_type: NodePort
+  sc_deployment_hostname_suffix: 
+  sc_ingress_tis_secret_name: "filenet-poc-tls"
+  sc_ingress_annotations:
+  // highlight-next-line
+    - nginx.ingress.kubernetes.io/proxy-body-size: "0"
+    - nginx.ingress.kubernetes.io/affinity: cookie
+    - nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    - nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    - nginx.ingress.kubernetes.io/secure-backends: "true"
+    - nginx.ingress.kubernetes.io/session-cookie-name: route
+    - nginx.ingress.kubernetes.10/session-cookie-hash: sha1
+    - kubernetes.io/ingress.class: nginx
+```
+
+When `nginx.ingress.kubernetes.io/proxy-body-size` is set to 0, this removes they payload limit size. With this change to the CR, we were able to create documents with content successfully. 
+
+#### **Summary**
```

Comment:
@Myron-Woods-IBM you dont need to bold `**summary** after establishing a headline in markdown using hashtags

Reply:
Updated and removed bold styling from headlines